main.go

```diff
@@ -57,8 +97,45 @@ func main() {
 
 	err = discord.Open()
 	utils.Assert("Error opening connection to Discord", err)
+
+	for i := 0; i < len(discord.State.Guilds); i++ {
+		currentguild := discord.State.Guilds[i]
+		println(currentguild.ID)
+		channellist, err := discord.GuildChannels(currentguild.ID)
+		utils.Assert("error retrieving channellist", err)
+		println(channellist)
+		var (
+			messagechannel string
+		)
+		for a := 0; a < len(channellist); a++ {
+			if channellist[a].Type == discordgo.ChannelTypeGuildText {
+				messagechannel = channellist[a].ID
+			}
+			if channellist[a].Name == "general" {
+				message, err := discord.ChannelMessageSend(channellist[a].ID, "Hello Fish!")
+				utils.Assert("error sending message", err)
+				println(message.ID)
+			}
+		}
+		for r := 0; r < len(currentguild.Roles); r++ {
+			currentrole := currentguild.Roles[r]
+			println(currentrole.Name)
+			println(currentrole.ID)
+			if currentrole.Name == "admin" {
+				println(messagechannel)
+			}
+		}
+		ownerchannel, err := discord.UserChannelCreate(currentguild.OwnerID)
+		utils.Assert("error creating channel", err)
+		message, err := discord.ChannelMessageSend(ownerchannel.ID, "Welcome to the LANFest Discord bot! You are registered as the owner of "+currentguild.Name+", so you will need to answer a few questions to complete setup. If you’d rather not, please type the discord ID of another admin-level access user (e.g. "+botname+botID+") to complete setup for you. Type y to continue.")
+		utils.Assert("error sending message", err)
+		println(message.ID)
```

Comment:
You're already doing this in the 'onReady' handler above (starting with line 52).  Still necessary to do it here as well?